kong/clustering/services/sync/rpc.lua

```diff
@@ -402,7 +402,7 @@ function sync_once_impl(premature, retry_count)
   local current_version = get_current_version()
   if current_version >= latest_notified_version then
     ngx_log(ngx_DEBUG, "version already updated")
-    return
+    return sync_handler() -- one more call to report CP the latest version synced
```

Comment:
Could you explain the reason? Why we should do this?

Reply:
This was the original behavior, where we rely on an extra call to tell the CP, that DP is on the latest version. Only with a "get_delta" call does CP gets the DP's current version

Reply:
I'm kind of impressed with the design, but not all of it, do you know where the documentation is?

Reply:
I think that CP don't care about DP's version, what it will cause in CP?

Reply:
I think UI needs this version.